### PR TITLE
Initial Commit for v1.1.0

### PR DESCRIPTION
**Pull request by cs-adarsh-gupta**
### Descriptions:
- New action: Close Incidents
- Debug logs added.
- Internal logic is changed for function Get Incidents.

#### Affected Actions:
- [ ] Close Incidents

#### UTCs:
- [ ] Connector installation verified
- [ ] Connector logo verified
- [ ] Check health verified
- [ ] Docs link verified
- [ ] Actions and Playbooks list verified
- [ ] Playbook tags verified
- [ ] All playbooks are in info mode verified
- [ ] All playbooks are in inactive mode verified
- [ ] Ingestion playbooks are verified
- [ ] Verified platform dependencies remain unaffected

### CODE REVIEW

File: docs/ProofpointTRAP.md [hunk 1]
```diff
@@ -87,8 +95,7 @@ The output contains the following populated JSON schema:
 <table border=1><thead><tr><th>Parameter</th><th>Description</th></tr></thead><tbody><tr><td>State</td><td>Select state of the incident
 </td></tr><tr><td>Created After Date</td><td>Retrieve incidents that were created after specified date, in ISO 8601 format (UTC)
 </td></tr><tr><td>Created Before Date</td><td>Retrieve incidents that were created before specified date, in ISO 8601 format (UTC)
-</td></tr><tr><td>Closed After Date</td><td>Retrieve incidents that were closed after specified date, in ISO 8601 format (UTC)
```
Comment (cs-utkarsh-surana) [inline docs/ProofpointTRAP.md:90]: Mention this under release notes.

File: docs/ProofpointTRAP.md [hunk 2]
```diff
@@ -87,8 +95,7 @@ The output contains the following populated JSON schema:
...
-</td></tr><tr><td>Cclosed Before Date</td><td>Retrieve incidents that were closed before specified date, in ISO 8601 format (UTC)
+</td></tr><tr><td>Email Message ID</td><td>Specify comma separated value of multiple message-ids.
```
Comment (cs-utkarsh-surana) [inline docs/ProofpointTRAP.md:102]: Add details about this parameter in release notes.

File: docs/ProofpointTRAP.md [hunk 3]
```diff
@@ -87,8 +95,7 @@ The output contains the following populated JSON schema:
 <table border=1><thead><tr><th>Parameter</th><th>Description</th></tr></thead><tbody><tr><td>State</td><td>Select state of the incident
 </td></tr><tr><td>Created After Date</td><td>Retrieve incidents that were created after specified date, in ISO 8601 format (UTC)
 </td></tr><tr><td>Created Before Date</td><td>Retrieve incidents that were created before specified date, in ISO 8601 format (UTC)
-</td></tr><tr><td>Closed After Date</td><td>Retrieve incidents that were closed after specified date, in ISO 8601 format (UTC)
-</td></tr><tr><td>Cclosed Before Date</td><td>Retrieve incidents that were closed before specified date, in ISO 8601 format (UTC)
```
Comment (cs-utkarsh-surana) [inline docs/ProofpointTRAP.md:91]: This as well.

File: proofpoint-trap/info.json [hunk 4]
```diff
@@ -1,15 +1,20 @@
 {
   "name": "proofpoint-trap",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "label": "Proofpoint TRAP",
-  "description": "Perform actions like Get incident details, Retrieve incidents, Update Incident, Add incident comments, Add user to incident, Update Incident Description, Update Incident Description, Close Incident, and Create Alert from JSON source using Proofpoint TRAP",
+  "description": "Perform actions like Get incident details, Retrieve incidents, Close Incidents and Create Alert from JSON source using Proofpoint TRAP",
   "publisher": "Fortinet",
```
Comment (cs-utkarsh-surana) [inline proofpoint-trap/info.json:6]: add contributor key

File: proofpoint-trap/utils.py [hunk 5]
```diff
@@ -19,18 +19,33 @@ def invoke_rest_endpoint(config, endpoint, method='GET', data=None, headers=None
...
         raise ConnectorError('Missing required parameters')
-    headers = {'accept': 'application/json', 'Authorization': apiToken}
-    url = '{server_address}{endpoint}'.format(server_address=server_address,endpoint=endpoint)
+    headers = {'Content-type': 'application/json', 'accept': 'application/json', 'Authorization': apiToken}
+    url = '{server_address}{endpoint}'.format(server_address=server_address, endpoint=endpoint)
     try:
         response = requests.request(method, url, verify=verify_ssl,
                                     data=json.dumps(data), headers=headers)
-        logger.debug('RESPONSE STATUS CODE = {}'.format(str(response.status_code)))
+        logger.error('RESPONSE STATUS CODE = {}'.format(str(response.status_code)))
```
Comment (cs-utkarsh-surana) [inline proofpoint-trap/utils.py]: set it to debug (both)

**Review by cs-utkarsh-surana — CHANGES_REQUESTED** [review]

File: proofpoint-trap/info.json (hunk 4 above)
Comment (cs-adarsh-gupta) [reply proofpoint-trap/info.json:6]: Contributor key here is not valid, as the contributor has left the organisation

**Pull request merged**